shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -194,6 +194,12 @@ bool FlutterTizenEngine::RunEngine(
         auto engine = reinterpret_cast<FlutterTizenEngine*>(user_data);
         auto message = engine->ConvertToDesktopMessage(*engine_message);
         engine->message_dispatcher->HandleMessage(message);
+
+        // Workarounds for handling engine restarts. Needs to be replaced by
+        // PlatformView::OnPreEngineRestart.
+        if (strncmp(message.channel, "flutter/isolate", 15) == 0) {
+          engine->RestartEngine();
+        }
```

Comment:
What is the exact purpose of the "flutter/isolate" channel? 
```FlutterDesktopMessage``` also has a channel and message. I think you should check the message too.
Lastly, Is this workaround a necessary?

Reply:
Message contains isolate's id like this:
```
isolates/880563550613439
```
Now that I see it, this workaround can't use this propose. So I'll revert it.